gptcache/manager/eviction.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, scalar_storage, vector_base, policy="LRU"):
         self._policy = policy
 
     def check_evict(self):
-        mark_count = self._scalar_storage.count(state=-1)
+        mark_count = self._scalar_storage.count()
```

Comment:
it will get the data which's state is not deleted, the mark count should be the count of the deleted data

Reply:
fixed